   ml_ops/sm-datazone_import/README.md

```diff
@@ -13,6 +13,7 @@ aws configure add-model --service-model file://resources/datazone-linkedtypes-20
 2. Create a federation role
 
 This role will be used by DataZone to launch the SageMaker Domain. See [BringYourOwnDomainResources.yml](.resources/BringYourOwnDomainResources.yml) for an example.
+If you are using multiple projects with multiple users, you will need to create a separate SageMaker Execution Role as well as Federation Role.
```

Comment:
- could be phrased better - "...you will need to create a separate SageMaker Execution role and User Profile for each user in each project and a separate Federation role per project."
- This premise of this statement is related to when a single SageMaker Domain is involved right? So something like "If you are using a single SageMaker domain across multiple projects with ...."

Reply:
Yes  - the second bullet is correct. Let me add that pretense and rephrase, good points.